recipes/hpp-gui/COPYING

```diff
@@ -0,0 +1,165 @@
+           GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
You only need to include the license files if they're not included in the source tarballs, as you're using Github releases they should be present.

Reply:
The repo of some of the recipe does not contain the file, some does. As they all point to the same organisation and global project. Is this approach ok with conda-forge or should we wait our next github release that will include file ?

Reply:
Where do the license files come from? This way is fine provided the files are "approved" by the upstream copyright holders.

Reply:
This is a common fille present in some of the package but not on some others. This is the common file on the project, I only patch the package that does not have one. But I will confirm with upstream before merge !